src/Tizen.NUI/src/internal/WebView/WebSettings.cs

```diff
@@ -102,6 +105,56 @@ public bool EnableWebSecurity
             }
         }
 
+        /// <summary>
+        /// Enable cache builer
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool EnableCacheBuilder
```

Comment:
According to comments from https://code.sec.samsung.net/confluence/display/GFX/WebView ,
Naming rule of properties is like xxxEnabled.

Reply:
Done